packages/@ourworldindata/grapher/src/stackedCharts/StackedUtils.ts

```diff
@@ -28,6 +28,30 @@ export const stackSeries = <PositionType extends StackedPointPositionType>(
     return seriesArr
 }
 
+// This method shifts up positive y values and shifts down negative y values of a Series with Points in place.
+export const stackSeriesInBothDirections = <
+    PositionType extends StackedPointPositionType,
+>(
+    seriesArr: readonly StackedSeries<PositionType>[]
+): readonly StackedSeries<PositionType>[] => {
+    seriesArr.forEach((series, seriesIndex) => {
+        if (!seriesIndex) return // The first series does not need to be shifted
+        series.points.forEach((point, pointIndex) => {
+            const pointsBelowThisOne = seriesArr
+                .slice(0, seriesIndex)
+                .map((s) => s.points[pointIndex])
+            const pointBelowThisOne =
+                point.value < 0
+                    ? last(pointsBelowThisOne.filter((p) => p.value < 0))
+                    : last(pointsBelowThisOne.filter((p) => p.value >= 0))
```

Comment:
```suggestion
                    ? findLast(pointsBelowThisOne, (p) => p.value < 0)
                    : findLast(pointsBelowThisOne, (p) => p.value >= 0)
```

... where [`findLast`](https://lodash.com/docs/4.17.15#findLast) is taken from lodash (it's currently not re-exported from `Util.ts`, but you can add it to the list).